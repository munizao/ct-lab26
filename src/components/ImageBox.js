import React from 'react';
import PropTypes from 'prop-types';
import styles from './App.css';
import Title from './Title';
import kittenImage from '../assets/kitten1.jpeg'

const ImageBox = (props) => (
  <div className={styles.ImageBox}>
    <div className="card-image">
      <img src={kittenImage} alt={props.name} />
    </div>
    <Title id={props.id} name={props.name}/>
  </div>
);

ImageBox.propTypes = {
  id: PropTypes.number,
  name: PropTypes.string
};

export default ImageBox;
