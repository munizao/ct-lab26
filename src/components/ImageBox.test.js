import React from 'react';
import { shallow } from 'enzyme';
import ImageBox from './ImageBox';

describe('ImageBox component', () => {
  it('renders ImageBox', () => {
    const wrapper = shallow(<ImageBox id="132" name="Gar's Mytholog" />);
    expect(wrapper).toMatchSnapshot();
  });
});
