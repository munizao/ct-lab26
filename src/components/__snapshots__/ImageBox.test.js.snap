// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageBox component renders ImageBox 1`] = `
<div
  className="ImageBox"
>
  <div
    className="card-image"
  >
    <img
      alt="Gar's Mytholog"
      src="https://rickandmortyapi.com/api/character/avatar/132.jpeg"
    />
  </div>
  <Title
    id="132"
    name="Gar's Mytholog"
  />
</div>
`;
